database/migrations/2020_05_04_100000_create_escaperoom_slots_table.php

```diff
@@ -20,8 +20,8 @@ public function up()
             $table->date('date')->index();
             $table->dateTime('start_at');
             $table->dateTime('end_at'); // Computed from room->duration
-            $table->dateTime('room_available_at'); // Time when the room is available for another slot. Automatically computed by adding room->occupied_time to slot->start_at
```

Comment:
This field should remain as `room_available_at` instead of being changed to `slot_available_at`

@sl0wik Can you update the Whimsical diagram?

Reply:
@drewroberts I I revert the changes as `slot_available_at` ... pending to the changes in Whimsical @sl0wik

Reply:
Can be called room_available_at because this is EscaperoomSlot not just a regular slot.

Reply:
Thanks!